lib/tasks/demo_data_for_development.rake

```diff
@@ -0,0 +1,67 @@
+        create(:event, program: program, title: 'Demo Unconfirmed Event', state: 'unconfirmed', abstract: 'This is a demo event instance in unconfirmed state.')
+        create(:event, program: program, title: 'Demo Confirmed Event', state: 'confirmed', abstract: 'This is a demo event instance in a confirmed state.')
+
+        first_scheduled_event = create(:event, program: program, title: 'first_scheduled_event', state: 'confirmed', abstract: 'This is a demo scheduled event instance.')
+        second_scheduled_event = create(:event, program: program, title: 'second_scheduled_event', state: 'confirmed', abstract: 'This is a demo scheduled event instance.')
+        multiple_speaker_event = create(:event, program: program, title: 'multiple_speaker_event', state: 'confirmed', abstract: 'This is a demo scheduled event instance having multiple speakers.')
+
+        create(:event_user, event: multiple_speaker_event, user: user1, event_role: "speaker")
+        create(:event_user, event: multiple_speaker_event, user: user2, event_role: "speaker")
+
+        conference_rooms = conference.venue.rooms
+
+        create(:event_schedule, event: first_scheduled_event, schedule: selected_schedule, start_time: conference.start_date + conference.start_hour.hours, room: conference_rooms.first)
+        create(:event_schedule, event: second_scheduled_event, schedule: selected_schedule, start_time: conference.start_date + conference.start_hour.hours + 15.minutes, room: conference_rooms.second)
+        create(:event_schedule, event: multiple_speaker_event, schedule: selected_schedule, start_time: conference.start_date + conference.start_hour.hours + 30.minutes, room: conference_rooms.third)
+        create(:event_schedule, event: first_scheduled_event, schedule: demo_schedule, start_time: conference.start_date + conference.start_hour.hours + 15.minutes, room: conference_rooms.third)
+        create(:event_schedule, event: second_scheduled_event, schedule: demo_schedule, start_time: conference.start_date + conference.start_hour.hours + 30.minutes, room: conference_rooms.third)
+        create(:event_schedule, event: multiple_speaker_event, schedule: demo_schedule, start_time: conference.start_date + conference.start_hour.hours, room: conference_rooms.first)
+
+        create(:registration, user: user1, conference: conference)
+        create(:registration, user: user2, conference: conference)
+    end
+
+    conference = create(:full_conference, title: 'Jangouts Demo Conference', short_title: 'jangouts', start_hour: 15, description: "This is a full conference demo instance happened in the past. It contains open cfp, venue/rooms, submitted talks by multiple speakers, partly confirmed talks and multiple schedules.")
+    program = conference.program
+    generate_events program, conference
+    conference.update_attributes(start_date: 7.days.ago, end_date: 1.day.ago)
```

Comment:
>I also think that it would be great that one of the conferences is not over yet, I mean that the end_date is not before the current day, but after.

I have made yast conference like this. :wink: